### DISCUSSION TITLE
Can I delete fork later ?

### DISCUSSION THREAD

**Discussion by anneranch** [blueman-project/blueman#2156] [Q&A] [answered]
Feel free to [open a GitHub issue](https://github.com/blueman-project/blueman/issues/new) to file bugs, or ask about anything you need help with.

Will fork include ALL prerequisites ?
Suggested answer 
yes 
no

If I fork the source code , how can I run / debug it ? 
Suggested answer 
.....run.... 

Can I delete fork later ? 
Suggested answer 
yes 
no

Thanks

**Comment by cschramm** [comment, marked as answer]
No, see [Dependencies.md](https://github.com/blueman-project/blueman/blob/main/Dependencies.md) for dependencies.

See [README.md#installing](https://github.com/blueman-project/blueman/blob/main/README.md#installing). You can run locally after `make`, e.g. as `BLUEMAN_SOURCE=1 python3 apps/blueman-manager`. As an alternative to make you can use meson / ninja and run from the build directory with `BLUEMAN_SOURCE=1` as well.

If you mean a GitHub fork, just delete your repository, see e.g. https://stackoverflow.com/questions/11302639/delete-forked-repo-from-github. If you mean a Git clone to your computer, well, remove it from your computer.
